<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="generator" content="pandoc" />
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <meta name="author" content="Abraham Gihawi" />
    <meta name="date" content="2022-04-22" />
    <title>Abraham Gihawi - Curriculum Vitae</title>

    
        <link href="Gihawi_CV_files/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
        <link href="Gihawi_CV_files/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />
        <link href="Gihawi_CV_files/paged-0.17.3/css/resume.css" rel="stylesheet" />
        <script src="Gihawi_CV_files/paged-0.17.3/js/config.js"></script>
        <script src="Gihawi_CV_files/paged-0.17.3/js/paged.js"></script>
        <script src="Gihawi_CV_files/paged-0.17.3/js/hooks.js"></script>
    
        <link rel="stylesheet" href="css/styles.css" type="text/css" />
        <link rel="stylesheet" href="css/academicons.css" type="text/css" />
        <link rel="stylesheet" href="css/index_files/font-awesome-5.1.0/css/all.css" type="text/css" />
    
  </head>
  <body>

<style type="text/css">
.pagedjs_page:not(:first-of-type) {
  --sidebar-width: 0rem;
  --sidebar-background-color: #ffffff;
  --main-width: calc(var(--content-width) - var(--sidebar-width));
  --decorator-horizontal-margin: 0.2in;
}
</style>
<div id="aside" class="section level1">
<h1>Aside</h1>
<div id="contact" class="section level2">
<h2>Contact</h2>
<ul>
<li><i class="fa fa-envelope"></i> <a href="mailto::a.gihawi@uea.ac.uk">AbrahamGihawi@hotmail.com</a></li>
<li><i class="fa fa-github"></i> <a href="https://github.com/Agihawi">Agihawi</a></li>
<li><i class="fa fa-twitter"></i> <a href="https://twitter.com/AbrahamGihawi">@AbrahamGihawi</a></li>
<li><i class="fa fa-phone"></i> +44 7793068331</li>
<li><i class="ai ai-orcid"></i> <a href="https://orcid.org/0000-0002-3676-5561">ORCID</a></li>
<li><i class="fab fa-linkedin-in"></i> <a href="https://www.linkedin.com/in/abraham-gihawi-887954a6/">LinkedIn</a></li>
<li><i class="ai ai-google-scholar"></i> <a href="https://scholar.google.co.uk/citations?user=CzHmHBYAAAAJ&amp;hl=en&amp;oi=ao">Google Scholar</a></li>
</ul>
</div>
<div id="skills" class="section level2">
<h2>Skills</h2>
<p><strong>Programming:</strong> R, Python, Unix</p>
<ul>
<li>Computational Pipelines</li>
<li>Microbial Bioinformatics</li>
<li>High Performance Computing</li>
<li>Dimensionality Reduction</li>
<li>Translational Cancer Research</li>
</ul>
</div>
<div id="conferences" class="section level2">
<h2>Conferences</h2>
<p>Posters:</p>
<p><a href="https://github.com/Agihawi/gel_summit_2022/blob/main/gel_summit_2022.pdf">Genomics England 2022</a> <br>
<a href="https://github.com/UEA-Cancer-Genetics-Lab/EACR_Bioinformatics_2021/blob/master/Poster.pdf">EACR Bioinformatics in Cancer, 2021</a> <br>
<a href="https://agihawi.github.io/ncrn_poster/">NCRN Symposium, 2020</a> <svg aria-hidden="true" role="img" viewBox="0 0 576 512" style="height:1em;width:1.12em;vertical-align:-0.125em;margin-left:auto;margin-right:auto;font-size:inherit;fill:gold;overflow:visible;position:relative;"><path d="M552 64H448V24c0-13.3-10.7-24-24-24H152c-13.3 0-24 10.7-24 24v40H24C10.7 64 0 74.7 0 88v56c0 35.7 22.5 72.4 61.9 100.7 31.5 22.7 69.8 37.1 110 41.7C203.3 338.5 240 360 240 360v72h-48c-35.3 0-64 20.7-64 56v12c0 6.6 5.4 12 12 12h296c6.6 0 12-5.4 12-12v-12c0-35.3-28.7-56-64-56h-48v-72s36.7-21.5 68.1-73.6c40.3-4.6 78.6-19 110-41.7 39.3-28.3 61.9-65 61.9-100.7V88c0-13.3-10.7-24-24-24zM99.3 192.8C74.9 175.2 64 155.6 64 144v-16h64.2c1 32.6 5.8 61.2 12.8 86.2-15.1-5.2-29.2-12.4-41.7-21.4zM512 144c0 16.1-17.7 36.1-35.3 48.8-12.5 9-26.7 16.2-41.8 21.4 7-25 11.8-53.6 12.8-86.2H512v16z"/></svg> <br>
<a href="https://github.com/Agihawi/Gel_Conference_Poster_1/blob/master/Posterdown.pdf">Genomics England, 2019</a> <br>
<a href="https://www.embo-embl-symposia.org/symposia/2018/EES18-09/index.html">EMBL Heidelberg, 2018</a></p>
<p>Talks:</p>
<p><a href="https://clinicalmetagenomics.org/teams/abraham-gihawi/">ICCMG5 2021</a> <br>
<a href="https://twitter.com/CancerInNorwich/status/1353738228625514500">NCRN, 2021</a> <br>
UEA PGR Conference, 2018</p>
</div>
<div id="references" class="section level2">
<h2>References</h2>
<p><a href="https://people.uea.ac.uk/d_brewer">Prof. Daniel Brewer</a> <a href="mailto:D.Brewer@uea.ac.uk" class="email">D.Brewer@uea.ac.uk</a></p>
<p><a href="https://people.uea.ac.uk/colin_cooper">Prof. Colin Cooper</a> <a href="mailto:Colin.Cooper@uea.ac.uk" class="email">Colin.Cooper@uea.ac.uk</a></p>
</div>
<div id="disclaimer" class="section level2">
<h2>Disclaimer</h2>
<p>Code available: <a href="https://github.com/Agihawi/Gihawi_CV">github.com/Agihawi/Gihawi_CV</a></p>
<p>Last updated: 2022-04-22</p>
</div>
</div>
<div id="main" class="section level1">
<h1>Main</h1>
<div id="title" class="section level2">
<h2>Abraham Gihawi</h2>
<div id="computational-biologist" class="section level3">
<h3>Computational Biologist</h3>
<p>Multidisciplinary bioinformatician with a keen interest in cancer and pathogen genomics. During my PhD, developed <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1819-8">containerised computational pipelines</a> to allow for the automated and reproducible retrieval of pathogenic sequences from within human tissue DNA sequencing data. Also formulated hypotheses as to how pathogens influence tumour biology by searching for <a href="https://doi.org/10.1016/j.euo.2022.03.006">clinical associations</a>.</p>
</div>
</div>
<div id="academia" class="section level2" data-icon="graduation-cap" data-concise="true">
<h2>Academia</h2>
<div id="senior-research-associate-in-bioinformatics" class="section level3">
<h3>Senior Research Associate in Bioinformatics</h3>
<p>PanProstate Cancer Group Bioinformatician</p>
<p>UEA</p>
<p>2022 - 2023</p>
<p>This role beginning July 2022 will entail harmonising and analysing clinical data from international sources within the Pan-Prostate Cancer Group. I am planning to develop and maintain databases to facilitate the robust dissemination of data to collaborators. These resources will be used to investigate the rich content of the PPCG including microbial composition.</p>
</div>
<div id="senior-bioinformatics-officer" class="section level3">
<h3>Senior Bioinformatics Officer</h3>
<p>Cancer Genetics and Metagenomics</p>
<p>UEA</p>
<p>2021 - Aug 2022</p>
<p>This position stemmed from a successful grant application using insights obtained during my PhD. This project uses whole genome and RNA sequencing data from the Pan-Prostate Cancer Group (PPCG) to identify how microbes might be associated with aggressive prostate cancer.</p>
</div>
<div id="phd-computational-biology" class="section level3">
<h3>PhD Computational Biology</h3>
<p>Thesis: <em>“Searching for Pathogens in Cancer Sequence Data”</em></p>
<p>UEA</p>
<p>2021 - 2017</p>
<p>Supervisors: Prof. Daniel Brewer, Dr. Ghanasyam Rallapalli, Dr. Richard Leggett, Dr. Rachel Hurst, Prof. Colin Cooper</p>
<p>In this position I developed benchmarked approaches to search for microbial nucleic acids within human tissue sequence data. I applied these pipelines to multiple datasets including Genomics England’s 100,000 Genomes Project. This has lead to characterising the microbial landscape of the tumour microenvironment. I have also investigated human and metagenomic transcriptomic data from exosomal fractions of urine from patients under investigation for prostate cancer. This has revealed some potentially impactful associations with clinical metadata and generated hypotheses regarding how pathogens interact with tumours. Additionally this work has lead to successful grant applications and intellectual property.</p>
<div id="technical-skills" class="section level4">
<h4>Technical Skills:</h4>
<p>Proficiency in various programming languages (R, Python and Unix), Developing and applying computational pipelines (Snakemake), handling large quantities of sequencing data, containerising software, reproducible research, microbial bioinformatics, high performance computing, dimensionality reduction, version control software, metagenomic assembly, assembly of bacterial isolates, prediction of metabolic pathways, human RNA differential expression, analysis of 16S ribosomal RNA sequencing data, phylogenetics, clustering algorithms, machine learning approaches (random forest, boruta), survival analysis, submission of sequence data to public archives</p>
</div>
<div id="other-skills" class="section level4">
<h4>Other Skills:</h4>
<p>Scientific writing, problem solving, critical thinking, project management and organisation, presentation skills, translational research, multidisciplinary collaboration.</p>
</div>
</div>
<div id="msc-cancer-therapeutics---distinction---80" class="section level3">
<h3>MSc, Cancer Therapeutics - Distinction - 80%</h3>
<p>Project: PHLDA1 in FGFR Inhibitor Resistant Cancer</p>
<p>QMUL</p>
<p>2017 - 2015</p>
<p>Supervisors: Prof. Richard Grose</p>
<p>In this course I delved into the depths of how we can leverage the underlying science of cancer for clinical benefit. I studied a range of subjects relating to cancer including: cancer biology ablative therapies, cancer pathology, cancer pharmacology, cancer prevention and screening, molecular diagnostics and therapeutics, drug development molecular diagnostics and therapeutics, molecular targeted therapies for haematological malignancies and genomic approaches to human disease. My project involved characterising <em>PHLDA1</em> which was shown to be the most significantly downregulated gene in fibroblast growth factor receptor (FGFR) inhibitor resistant endometrial cancer.</p>
<div id="technical-skills-1" class="section level4">
<h4>Technical Skills:</h4>
<p>Mammalian cell culture, western blotting, 3D organotypic culture models, clonogenic assays, efficient laboratory work, analysing immunofluorescence images, searching for synergistic relationship between candidate drugs.</p>
</div>
<div id="other-skills-1" class="section level4">
<h4>Other Skills:</h4>
<p>Time management, Student representative for the course, conveying the concerns of both on-site students as well as remote leaners to faculty staff.</p>
</div>
</div>
<div id="bsc-hons-biomedical-science---69" class="section level3">
<h3>BSc (hons) Biomedical Science - 69%</h3>
<p>Project: A Portable Test for Bacteria &amp; Urobilinogen in Urine</p>
<p>Lincoln</p>
<p>2014 - 2011</p>
<p>Supervisor: Dr Jose Gonzalez-Rodriguez</p>
<p>This course, accredited by the Institute of Biomedical Science (IBMS) provided me with a base level of knowledge in the biomedical field. I read several topics including: Pharmacology, Anatomy &amp; Physiology, Biochemistry, Molecular Biology, Genetics, Health &amp; Disease, Microbiology, Immunology, General Good Laboratory Practice, Research Skills. Seeking to develop my scientific independence, I selected the research topic “Developing a Portable Assay for the Detection of Bacteria &amp; Urobilinogen in Urine” for my final year project. In this project I developed assays for bacteria and urobilinogen in urine and investigated whether mobile phone cameras could accurately quantify these analytes.</p>
<div id="technical-skills-2" class="section level4">
<h4>Technical Skills:</h4>
<p>Good laboratory practice, research methods, microbiological culture, gram staining, independent scientific research, microtomy, microscopy, blood cell morphology, tissue staining and histology, electrophoresis, pipetting, fluorescent <em>in situ</em> hybridization, latex agglutination assays.</p>
</div>
</div>
</div>
<div id="experience" class="section level2" data-icon="laptop">
<h2>Experience</h2>
<div id="medical-laboratory-assistant" class="section level3">
<h3>Medical Laboratory Assistant</h3>
<p>Royal Sussex County Hospital</p>
<p>Brighton</p>
<p>2017 - 2015</p>
<p>In this position I was responsible for implementing and conducting laboratory processing of samples for research studies and clinical trials. My day to day duties involved collaborating with a wide range of clinical and non-clinical staff, maintaining laboratory equipment sample processing such as centrifugation, blood smears, flow cytometry and isolation of peripheral blood mononuclear cells. One of the ways this position stimulated my interest in cancer was by carrying out a trial for the Endopredict <svg aria-hidden="true" role="img" viewBox="0 0 512 512" style="height:1em;width:1em;vertical-align:-0.125em;margin-left:auto;margin-right:auto;font-size:inherit;fill:currentColor;overflow:visible;position:relative;"><path d="M285.363 207.475c0 18.6-9.831 28.431-28.431 28.431h-29.876v-56.14h23.378c28.668 0 34.929 8.773 34.929 27.709zM504 256c0 136.967-111.033 248-248 248S8 392.967 8 256 119.033 8 256 8s248 111.033 248 248zM363.411 360.414c-46.729-84.825-43.299-78.636-44.702-80.98 23.432-15.172 37.945-42.979 37.945-74.486 0-54.244-31.5-89.252-105.498-89.252h-70.667c-13.255 0-24 10.745-24 24V372c0 13.255 10.745 24 24 24h22.567c13.255 0 24-10.745 24-24v-71.663h25.556l44.129 82.937a24.001 24.001 0 0 0 21.188 12.727h24.464c18.261-.001 29.829-19.591 21.018-35.587z"/></svg> prognostic qPCR expression assay for patients with HER2- ER+ breast cancer.</p>
</div>
<div id="health-advisor-nhs-111" class="section level3">
<h3>Health Advisor NHS 111</h3>
<p>Care UK</p>
<p>Dorking</p>
<p>2015 - 2014</p>
<p>In this position I utilised my knowledge to assist patients with their medical concerns. I triaged them over the telephone and directed them towards the appropriate medical services. This post allowed me to witness the human element of medical care. I handled a wide range of clinical events ranging from physical trauma to mental health crises, safeguarding concerns, urinary tract infections to symptoms of metastatic cancer.</p>
</div>
<div id="nhs-experience-placements" class="section level3">
<h3>NHS Experience Placements</h3>
<p>Princess Diana &amp; Scunthorpe General</p>
<p>Grimsby &amp; Scunthorpe</p>
<p>2014 - 2013</p>
<p>These work experience placements allowed me to witness many of the different aspects of patient care by assisting staff in a range of departments including: Radiology, Nuclear Medicine, Cardiology and Pathology (Biochemistry, Haematology, Transfusion, Andrology, Virology). This was an incredibly useful set of experiences that I feel has given me a holistic perspective on patient care.</p>
</div>
</div>
<div id="first-author-publications" class="section level2" data-icon="book">
<h2>First Author Publications</h2>
<div id="microbiomes-of-urine-and-the-prostate-are-linked-to-human-prostate-cancer-risk-groups" class="section level3">
<h3><i class="ai ai-open-access"></i> Microbiomes of Urine and the Prostate Are Linked to Human Prostate Cancer Risk Groups</h3>
<p>European Urology Oncology</p>
<p>DOI: <a href="https://doi.org/10.1016/j.euo.2022.03.006">10.1016/j.euo.2022.03.006</a></p>
<p>2022</p>
<p>Rachel Hurst, Emma Meader, <strong>Abraham Gihawi</strong> Ghanasyam Rallapalli… Colin Cooper.</p>
</div>
<div id="sepath-benchmarking-the-search-for-pathogens-in-human-tissue-whole-genome-sequence-data-leads-to-template-pipelines" class="section level3">
<h3><i class="ai ai-open-access"></i> SEPATH: benchmarking the search for pathogens in human tissue whole genome sequence data leads to template pipelines</h3>
<p>Genome Biology</p>
<p>DOI: <a href="https://doi.org/10.1186/s13059-019-1819-8">10.1186/s13059-019-1819-8</a></p>
<p>2019</p>
<p><strong>Abraham Gihawi</strong>, Ghanasyam Rallapalli, Rachel Hurst… Daniel S. Brewer</p>
</div>
</div>
<div id="additional-publications" class="section level2" data-icon="book">
<h2>Additional Publications</h2>
<div id="potential-for-diagnosis-of-infectious-disease-from-the-100000-genomes-project-metagenomic-dataset-recommendations-for-reporting-results" class="section level3">
<h3><i class="ai ai-open-access"></i> Potential for Diagnosis of Infectious Disease from the 100,000 Genomes Project Metagenomic Dataset: Recommendations for Reporting Results</h3>
<p>Wellcome Open Research</p>
<p>DOI: <a href="https://doi.org/10.12688/wellcomeopenres.15499.1">10.12688/wellcomeopenres.15499.1</a></p>
<p>2019</p>
<p>Gkikas Magiorkinis… <strong>Abraham Gihawi</strong>… Colin Cooper.</p>
</div>
</div>
<div id="grants" class="section level2" data-icon="money">
<h2>Grants</h2>
<div id="unraveling-how-bacterial-infection-drives-the-development-of-aggressive-prostate-cancer---prostate-cancer-uk" class="section level4">
<h4>“Unraveling how bacterial infection drives the development of aggressive prostate cancer” - Prostate Cancer UK</h4>
<p>I was a co-applicant alongside Prof. Daniel Brewer for this grant to provide financial support to fund a Senior Bioinformatics Officer for 18 months. This grant is responsible for my current position. Value: £77,925.</p>
</div>
<div id="the-role-of-bacterial-infection-in-the-development-of-human-prostate-cancer---bob-champion-cancer-trust" class="section level4">
<h4>“The Role of Bacterial Infection in the development of Human Prostate Cancer” - Bob Champion Cancer Trust</h4>
<p>I was Co-Applicant alongside Dr Rachel Hurst, Prof. Daniel Brewer and Prof. Colin Cooper. This grant finances laboratory investigations into the role of bacterial infection in the development of human prostate cancer. Bob Champion Cancer Trust. Value: £87,634.</p>
</div>
</div>
<div id="conferences-and-presentations" class="section level2" data-icon="file-powerpoint">
<h2>Conferences and Presentations</h2>
<p>Presenting scientific data is something that I find valuable and I have been fortunate to be able to present to a diverse range of audiences in recent years. This includes planning and preparing outreach to schools, producing award winning scientific posters and I was an invited speaker at the <a href="https://clinicalmetagenomics.org/teams/abraham-gihawi/">5th International Conference on Clinical Metagenomics</a> held in Geneva. Other international conferences I have attended and presented posters at include: The Human Microbiome, European Molecular Biology Laboratory Heidelberg (2018), and Bioinformatics in Cancer, virtual conference (2021)</p>
<p>One of my most successful presentations won a runner-up <a href="https://twitter.com/CancerInNorwich/status/1303009991255887873">award</a> for best poster after producing an <a href="https://t.co/aX1m9UiomG?amp=1">interactive poster</a> (code available on <a href="https://github.com/Agihawi/ncrn_poster">GitHub</a>) and giving a talk on the content to the audience. The posters were also hosted online which attracted a <a href="https://padlet.com/swest116/NCRNposter">wide range of participation and engagement</a>.</p>
<p>Other presentations I have given include a <a href="https://github.com/Agihawi/Gel_Conference_Poster_1/blob/master/Posterdown.pdf">poster presentation</a> (code available on <a href="https://github.com/Agihawi/Gel_Conference_Poster_1">GitHub</a> for a Genomics England research symposium, which included experienced scientists and project participants. Additionally, I have given talks and presented posters at our local university’s postgraduate researcher events as well as presenting frequently within my laboratory group meetings and departments bioinformatics meetings. In these bioinformatics meetings I have instigated programming challenges which I host fortnightly to enable us to discuss various aspects of scientific coding and problem solving.</p>
</div>
<div id="other" class="section level2" data-icon="balance-scale">
<h2>Other</h2>
<p><strong>Peer Review:</strong> Genome Biology (BMC), Genomics (Elsevier), Bioinformatics Advances (Oxford University Press)</p>
</div>
</div>

<script>
(function() {
  var i, s, j, el, els;
  var create_el = function(type, className, content, isHTML) {
    var el = document.createElement(type);
    if (className) el.className = className;
    if (content) {
      if (isHTML) {el.innerHTML = content;} else {el.innerText = content;}
    }
    return el;
  }

  // replace h2 title with h1
  var title = document.getElementById('title').querySelector('h2');
  title.parentNode.replaceChild(create_el('h1', false, title.innerHTML, true), title);

  // add the class .aside to #aside
  el = document.getElementById('aside');
  if (el) el.className += ' aside';

  // tweak class names of sections, and add icons
  els = document.getElementById('main').querySelectorAll('.level2');
  for (i = 0; i < els.length; i++) {
    el = els[i];
    if (i === 0 && el.id === 'title') continue;
    el.className += ' main-block';
    if (el.dataset['concise']) el.className += ' concise';
    // if there is no icon, add an icon:
    if (el.firstElementChild.firstChild && el.firstElementChild.firstChild.nodeName !== 'I') {
      s = el.dataset['icon'] || 'bookmark';
      el.firstElementChild.insertBefore(create_el('i', 'fa fa-' + s), el.firstElementChild.firstChild);
    }
  }

  // tweak class names of blocks in sections
  els = document.getElementById('main').querySelectorAll('.level3');
  for (i = 0; i < els.length; i++) {
    el = els[i];
    if (el.parentNode.id === 'title') continue;
    el.className += ' blocks';
    el.innerHTML = '<div class="date"></div>' + '<div class="decorator"></div>' +
      '<div class="details"><header></header><div></div></div>' + el.innerHTML;
    var ps = el.querySelectorAll('p');
    // move the date paragraph to the date div
    if (ps.length >= 3) {
      s = ps[2].innerText;
      s = s === 'N/A' ? [] : s.split(' - ');
      el.removeChild(ps[2]);
      for (j = 0; j < s.length && j < 2; j++) {
        el.children[0].appendChild(create_el('span', false, s[j]))
      }
    }
    // move title, description, location to the details div
    (function(h) {
      h.appendChild(el.children[3]);
      var p = el.children[3];  // description
      if (p.innerText !== 'N/A') {
        h.appendChild(create_el('span', 'place', p.innerHTML, true));
      }
      el.removeChild(p);
      p = el.children[3];  // location
      if (p.innerText !== 'N/A') {
        s = create_el('span', 'location', p.innerHTML, true);
        s.innerHTML = '<i class="fa fa-map-marker-alt"></i> ' + s.innerHTML;
        h.appendChild(s);
      }
      el.removeChild(p);
    })(el.children[2].firstElementChild);
    // move the rest of content in a block into the last div of the details div
    s = el.children[2];
    while (j = s.nextSibling) {
      if (j.className === 'aside') { s = j; continue; }
      el.children[2].lastElementChild.append(j);
    }
  }
})();
</script>


  </body>
</html>
